services/ui-src/src/verbiage/forms/adminBannerForm.ts

```diff
@@ -152,6 +157,7 @@ export const form = {
     {
       id: "abf-endDateYear",
       type: "child",
+      props: null,
```

Comment:
Hmm I wish we didn't have to add these. Is there a way around it? It creates noise in these files, unless I'm not understanding the benefit.